website/pages/docs/platform/k8s/injector/examples.mdx

```diff
@@ -19,12 +19,17 @@ using higher level constructs such as deployments, jobs or statefulsets.
 Before applying Vault Agent injection annotations to pods, the following requirements
 should be satisfied:
 
-- the master Kubernetes nodes can receive traffic from worker nodes on port `8080`,
+- the master Kubernetes nodes can connect to worker nodes on port `8080`,
```

Comment:
I like this, definitely more clear!

Reply:
This is a two-way policy setup right, i.e. the master nodes needs ingress policy from the worker nodes and the workers needs egress policy to the master nodes?